package.json

```
@@ -2,7 +2,7 @@
   "name": "size-report",
   "version": "0.1.0",
   "description": "Generate size report for pull request",
-  "author": "OpenThread Authors <openthread-users@googlegroups.com>",
+  "author": "OpenThread Authors https://github.com/openthread/openthread",
```

```suggestion
  "author": "OpenThread Authors (https://github.com/openthread/openthread)",
```

See: https://docs.npmjs.com/cli/v8/configuring-npm/package-json#people-fields-author-contributors